docs/xtrace.md

```diff
@@ -261,9 +278,15 @@ The following table lists all the tracepoint groups (`<group>`). Each group is a
 |Component name or names     |  Group name       |  Description                                                                                                              |
 |----------------------------|-------------------|---------------------------------------------------------------------------------------------------------------------------|
 |**j9mm**                    | `gclogger`        | A set of tracepoints that record each garbage collection cycle. Equivalent to `-verbose:gc` output                        |
```

Comment:
This also applies to `omrmm`. The list of groups is incomplete; I found this list (in OMR and OpenJ9):
```
	adaptivethread
	ageblending
	aging
	allocate
	alloclarge
	allocthreshold
	arraylet
	bytecodeMethods
	cardscrubbing
	checkjni
	checkvm
	classinit
	compact
	compiledMethods
	concurrent
	copyforwardscheme
	cuda
	darkMatterComparison
	dispatcher
	double_map
	dynamiccollectionset
	exvmaccess
	file
	gclogger
	heap
	j9file
	j9filestrean
	j9gs
	j9heap
	j9hypervisor
	j9mem
	j9mmap
	j9ri
	j9shared
	j9shmem
	j9shsem
	j9sig
	j9signal
	j9sl
	j9sock
	j9sysinfo
	j9syslog
	j9virt_rwlock
	j9vmem
	kickoff
	loaresize
	markdelegate
	mem
	methodArguments
	mmap
	nativeMethods
	nlsmessage
	omrfilestream
	omrshared
	omrshmem
	omrshsem
	omrsock
	oracle
	OSCache
	parallel
	percolate
	perfmon
	reclaim
	rememberedset
	resize
	runtimeexec
	scavenge
	scavenger
	signal
	sl
	stackremember
	staticMethods
	stringtable
	sysinfo
	syslog
	tarok
	testset1
	throw
	unload
	verboseclass
```